net-vpn/zerotier/files/zerotier-1.2.4-ZT_DEFAULT_HOME_PATH.patch

```diff
@@ -0,0 +1,22 @@
+https://github.com/zerotier/ZeroTierOne/pull/506
+
+--- a/osdep/OSUtils.cpp
++++ b/osdep/OSUtils.cpp
+@@ -363,6 +363,9 @@ std::vector<std::string> OSUtils::split(const char *s,const char *const sep,cons
+ 
+ std::string OSUtils::platformDefaultHomePath()
+ {
++#ifdef ZT_DEFAULT_HOME_PATH
++	return ZT_DEFAULT_HOME_PATH;
```

Comment:
Hm, can't you use the `#`-magic to make it work as a string instead of requiring explicit quoting?